src/Dev/FunctionalTest.php

```diff
@@ -66,7 +66,7 @@ class FunctionalTest extends SapphireTest implements TestOnly
         /**
          * Set this to true on your sub-class to use the draft site by default for every test in this class.
          *
-         * @deprecated 4.2.0:5.0.0 Use ?stage=Stage in your ->get() querystring requests instead
+         * @deprecated 4.2.0 Use ?stage=Stage in your ->get() querystring requests instead
```

Comment:
No, however specifically for this property we do not need to, because the only place it's used is as a conditional check for `useDraftSite()`, which itself is deprecated.